devilizer/visualizer/display_items.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+def display_items(in_lst: list, current: int) -> str:
```

Comment:
More accurate typehints
```suggestion
def display_items(in_lst: list[int], current: int) -> str:
```